code/example-project/app.js

```diff
@@ -0,0 +1,55 @@
+var path = require('path');
+
+var apos = require('apostrophe')({
+  shortName: 'final-test',
+
+  // See lib/modules for basic project-level configuration of our modules
+  // responsible for serving static assets, managing page templates and
+  // configuring user accounts.
+
+  modules: {
+
+    // Apostrophe module configuration
+
+    // Note: most configuration occurs in the respective
+    // modules' directories. See lib/apostrophe-assets/index.js for an example.
+    
+    // However any modules that are not present by default in Apostrophe must at
+    // least have a minimal configuration here: `moduleName: {}`
+
+    // If a template is not found somewhere else, serve it from the top-level
+    // `views/` folder of the project
+
+    'apostrophe-templates': { viewsFolderFallback: path.join(__dirname, 'views') },
+
+    'link-widgets': {},
+    'foods': {},
+    'foods-widgets': {
+        extend: 'apostrophe-pieces-widgets'
+         },
```

Comment:
Formatting is inconsistent. Not sure if you intend to ship this, you mentioned being unsure about it.